Packs/Phishing/Playbooks/Process_Email_-_Generic_v2_README.md

```diff
@@ -54,8 +59,10 @@ This playbook does not use any integrations.
 | Thread-Topic | The value of the "Thread-Topic" header which holds the original email subject, needed for forwarded email scenarios. It is passed as an input to the "Get Original Email - Generic v2" playbook to use in the relevant sub-playbooks. | incident.emailsubject | Optional |
 | EmailBrand | If this value is provided, only the relevant playbook runs. If no value is provided, all sub-playbooks are run.<br/>Possible values:<br/>- Gmail<br/>- EWS v2<br/>- MicrosoftGraphMail<br/>- EmailSecurityGateway<br/>Choosing the EmailSecurityGateway executes the following if enabled: - FireEye EX \(Email Security\) - Proofpoint TAP - Mimecast. |  | Optional |
 | EmailFileToExtract | Reported emails and emails retrieved during playbook execution can contain multiple nested email files. For example, an EML nested inside another EML file.<br/>If multiple level files are detected, this field determines which file represents the phishing email.<br/><br/>For example:<br/>User1 receives an email from Attacker. User1 attaches the email as an EML file and sends the email to User2.<br/>User2 also attaches that email as a file, and reports it as phishing. In this case, the phishing email would be the "inner file" \(as opposed to "outer file"\).<br/><br/>Possible values are: Inner file, Outer file, All files.<br/>Inner file: The file at the deepest level is parsed. If there is only one file, that file is parsed.<br/>Outer file: The file at the first level is parsed.<br/>All files: All files are parsed. Do not use this option in the phishing playbook, as there should only be one phishing email per playbook run. | Inner file | Optional |
+| UseOldHTMLFields | This input is used to preserve backward-compatibility. It determines whether the playbook should set email fields that are no longer being used in the out-of-the-box content.<br/>If set to True, the playbook will save data into the the "Email Body HTML" and "Rendered HTML" incident fields as it did before.<br/>If set to False, the playbook will not save data into those fields, and will simply be using the Email HTML field instead.<br/>If you are ingesting large emails which are causing issues with large amounts of data being saved into incident fields, you should set the value to False.<br/>We recommend setting the value to False unless you are certain that you need the "Email Body HTML" and "Rendered HTML" incident fields. | True | Optional |
```

Comment:
Maybe this is none of my business, but...
If we recommend setting this value to False, then shouldn't the default value be False?

Reply:
That's a good observation. However, this is intentional, as we didn't want to break backward-compatibility for existing users, so we recommend them to do it knowingly.